### DISCUSSION TITLE
Remix, tsyringe, and reflect-metadata

### DISCUSSION THREAD

**Discussion by AmirL** [remix-run/remix#5979] [Q&A]
Hi,

I'm trying to use tsyringe in my Remix project. Tsyringe works well without Remix, but it doesn't work with Remix. At least in dev mode.

I've added 
``` 
import 'reflect-metadata';
```
to the top in app/entry.server.tsx
but getting error "TypeInfo not known for ...". It seems it requires that import reflect-data should be exactly at the first line, but no idea how to do it, with "remix dev" command. Any ideas how to solve it?

**Comment by yukiyamazaki** [comment]
me too.
